enos/enos.py

```diff
@@ -470,14 +470,17 @@ def ssh_tunnel(env=None, **kwargs):
 
 
 @enostask("""
-usage: enos tc [--test] [-vv|-s|--silent]
+usage: enos tc [--test] [-e ENV|--env=ENV] [-vv|-s|--silent]
 
 Enforce network constraints
 
 Options:
   --test   Test the rules by generating various reports
+  -e ENV --env=ENV     Path to the environment directory. You should
+                       use this option when you want to link a specific
+                       experiment [default: %s].
   -h --help                 Show this help message.
```

Comment:
I think we should:
* sort the option list alphabetically (i.e. `--env`, `--help`, `--test`) similarly to other help messages; 
* align the description of `--test` and `--help` with the one of `--env`;
* add a comma at the end of the description of `--test`.